plugins/routed-eni/driver/driver_test.go

```diff
@@ -503,7 +503,7 @@ func TestSetupPodNetwork(t *testing.T) {
 	mockHostVeth.EXPECT().Attrs().Return(mockLinkAttrs)
 	//add host route
 	mockHostVeth.EXPECT().Attrs().Return(mockLinkAttrs)
-	mockNetLink.EXPECT().RouteAdd(gomock.Any()).Return(nil)
+	mockNetLink.EXPECT().RouteReplace(gomock.Any()).Return(nil)
```

Comment:
Should we add another RouteReplace to show that it works if it exists?

Reply:
We should improve the tests over all, but right now I'm ok with just making sure that it gets called.